autoscale_cloudroast/test_repo/autoscale/system/policies/test_system_execute_updated_policies.py

```diff
@@ -122,7 +122,7 @@ def test_update_policy_desired_capacity_over_25(self):
             .format(upd_policy_to_desired_capacity_execute, self.group.id))
         self.check_for_expected_number_of_building_servers(
             group_id=self.group.id,
-            expected_servers=26)
+            expected_servers=26, time_scale=False)
```

Comment:
I guess a comment here would be nice, explaining why we're disabling time scale on this mimic-using test